packages/casting/src/iterable.js

```diff
@@ -1,47 +1,6 @@
 import { E, Far } from '@endo/far';
-import { makeNotifier } from '@agoric/notifier';
 
-/**
- * @template T
- * @param {ERef<Notifier<T>>} notifier
- * @returns {ConsistentAsyncIterable<T>}
- */
-export const makeNotifierIterable = notifier =>
-  makeNotifier(E(notifier).getSharableNotifierInternals());
-
-/**
- * TODO: Remove this function when we have an @endo/publish-kit that suppports pull topics
- *
- * @template T
- * @param {ERef<PublicationRecord<T>>} tailP
- * @returns {AsyncIterator<T>}
- */
-const makeSubscriptionIterator = tailP => {
-  // To understand the implementation, start with
-  // https://web.archive.org/web/20160404122250/http://wiki.ecmascript.org/doku.php?id=strawman:concurrency#infinite_queue
-  return Far('SubscriptionIterator', {
-    next: async () => {
-      const resultP = E.get(tailP).head;
-      tailP = E.get(tailP).tail;
-      return resultP;
-    },
-  });
-};
-
-/**
- * TODO: Remove this function when we have an @endo/publish-kit that suppports pull topics
- *
- * @template T
- * @param {ERef<Subscription<T>>} subscription
- * @returns {ConsistentAsyncIterable<T>}
- */
-export const makeSubscriptionIterable = subscription =>
-  harden({
-    [Symbol.asyncIterator]: () =>
-      makeSubscriptionIterator(
-        E(subscription).getSharableSubscriptionInternals(),
-      ),
-  });
+export { subscribeEach, subscribeLatest } from '@agoric/notifier/subscribe.js';
```

Comment:
Why do we need deep imports? If notifier needs to export different things to different importers, could the notifier package use an `"exports":` section in its package.json to declare explicit importable module names? We're already doing this for some endo packages.

Reply:
`"exports":` is just enforcement for "deep imports".  Thanks for mentioning it; I will add that enforcement!